k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -32,30 +32,70 @@ max_consul_inst=3
 max_kafka_inst=3
 num_openldap_replicas=0 # Default the number of actual openldap instances
 num_worker_nodes=0
+not_ready_node_list=[]
+not_ready_node_count=0
 # Create a file consist of a list of node info and up to 'max_openldap_inst'
 # number of nodes. This file is used by OpenLDAP helm chart and will be deleted
 # at the end of this script.
 while IFS= read -r line; do
-    IFS=" " read -r -a node_name <<< "$line"
-    if [[ "$node_name" != "NAME" ]]; then
-        output=$(kubectl describe nodes $node_name | grep Taints | grep NoSchedule)
-        if [[ "$output" == "" ]]; then
-            node_list_str="$num_worker_nodes $node_name"
-            num_worker_nodes=$((num_worker_nodes+1))
-
-            if [[ "$num_worker_nodes" -le "$max_openldap_inst" ]]; then
-                num_openldap_replicas=$num_worker_nodes
-                node_list_info_path=$(pwd)/cortx-cloud-3rd-party-pkg/openldap/node-list-info.txt
-                if [[ -s $node_list_info_path ]]; then
-                    printf "\n" >> $node_list_info_path
-                fi
-                printf "$node_list_str" >> $node_list_info_path
+    IFS=" " read -r -a my_array <<< "$line"
+    node_name="${my_array[0]}"
+    node_status="${my_array[1]}"
+    if [[ "$node_name" == "NAME" && "$node_status" == "STATUS" ]]; then
```

Comment:
This is fine as-is, but you can add the `--no-headers` flag to your `kubectl get` commands and that will prevent this from showing up in the response to be parsed.

Reply:
cool, I didn't know that